cogboard-app/src/main/resources/initData/endpoints.json

```diff
@@ -13,6 +13,20 @@
       "publicUrl": "http://api-mocks:8080/zabbix/api_jsonrpc.php",
       "credentials": "credential1",
       "id": "endpoint3"
+    },
+    {
+      "id": "endpoint4",
+      "label": "Github",
+      "publicUrl": "https://github.com/",
+      "url": "https://github.com/",
+      "credentials": ""
+    },
+    {
+      "id": "endpoint5",
+      "label": "Bitbucket Cognifide",
+      "publicUrl": "https://bitbucket.cognifide.com/",
+      "url": "https://bitbucket.cognifide.com/",
+      "credentials": ""
```

Comment:
Remove this and instead add support to our Api Mocks - and mimic a response there. Please make sure you are not providing any sensitiva data.